server/models/Label.js

```diff
@@ -0,0 +1,29 @@
+import { Model } from 'objection';
+import objectionUnique from 'objection-unique';
+
+const unique = objectionUnique({ fields: ['name'] });
+
+export default class Label extends unique(Model) {
+  static get tableName() {
+    return 'labels';
+  }
+
+  static get pickJsonSchemaProperties() {
+    return true;
+  }
```

Comment:
There are two different styles in the on how to write this method.
```
  static get pickJsonSchemaProperties() {
    return true;
  }
```
and 

```
   static pickJsonSchemaProperties = true;
```

Let's choose one

Reply:
done